packages/material-ui/src/Typography/Typography.test.js

```diff
@@ -50,17 +44,30 @@ describe('<Typography />', () => {
 
     expect(container.firstChild).to.have.class(classes.alignCenter);
   });
-  ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'subtitle1', 'body2', 'body1', 'caption', 'button'].forEach(
-    (variant) => {
-      it(`should render ${variant} text`, () => {
-        // @ts-ignore literal/tuple type widening
-        const { container } = render(<Typography variant={variant}>Hello</Typography>);
-
-        expect(classes[variant] != null).to.equal(true);
-        expect(container.firstChild).to.have.class(classes[variant]);
-      });
-    },
-  );
+  [
+    'h1',
+    'h2',
+    'h3',
+    'h4',
+    'h5',
+    'h6',
+    'subtitle1',
+    'body2',
+    'body1',
+    'caption',
+    'button',
+    'overline',
+  ].forEach((variant) => {
+    it(`should render ${variant} text`, () => {
+      // @ts-ignore literal/tuple type widening
```

Comment:
This was added before we had `@ts-expect-error`.

`@ts-ignore` will stay forever unless we revisit.
`@ts-expect-error` will throw once there's no longer an error.

If it's a limitation of TS: `@ts-expect-error`, otherwise type casting.

Reply:
Let's leave the `@ts-ignore`. I'll look at them in a follow-up and ping you to explain the problem in more detail.

Reply:
Sure that would be helpful. Thanks!